orchestra/todos/api.py

```diff
@@ -6,7 +6,8 @@
 
 from orchestra.models import TodoListTemplate
 from orchestra.models import Todo
-from orchestra.models import Task
+from orchestra.models import Project
+from orchestra.utils.view_helpers import get_step_by_project_id_and_step_slug
```

Comment:
view_helpers is a funny module name in the sense that it implies things aren't usable elsewhere (not true---you use this function in serializers :))